sections/implementation/i18n.tex

```diff
@@ -0,0 +1,37 @@
+\section{Translations}
+To have a scalable application it is necessary to easily be able to translate the text to different languages.
+For this purpose we have used i18next, which is an internationalization framework for JavaScript.
+It is supported for many different frameworks like React, React Native, .net, Angular and many more \cite{react-i18next}.
+Instead of hardcoding the text on every page, we use the component withTranslations and gets the function t.  
```

Comment:
```suggestion
Instead of hardcoding the text on every page, we use the component \texttt{withTranslations} to get the function \texttt{t] for translation purposes.  
```